examples/quickstart/jupyter-notebooks/api-tutorial.ipynb

```diff
@@ -0,0 +1,480 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5e51170e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# GET \n",
+    "endpoint = \"/status/health\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"GET\"\n",
+    "\n",
+    "payload = {}\n",
+    "headers = {}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "\n",
+    "print(\"\\033[1mResponse\\033[0m: \" + response.text)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1de51db8-4c51-4b7e-bb3b-734ff15c8ab3",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Ingest data\n",
+    "\n",
+    "Now that you've confirmed that your cluster is up and running, you can start ingesting data. There are different ways to ingest data based on what your needs are. For more information, see [Ingestion methods](https://druid.apache.org/docs/latest/ingestion/index.html#ingestion-methods).\n",
+    "\n",
+    "This tutorial uses the multi-stage query (MSQ) task engine and its `sql/task` endpoint to perform SQL-based ingestion. The `/sql/task` endpoint accepts [SQL requests in the JSON-over-HTTP format](https://druid.apache.org/docs/latest/querying/sql-api.html#request-body) using the `query`, `context`, and `parameters` fields\n",
+    "\n",
+    "To learn more about SQL-based ingestion, see [SQL-based ingestion](https://druid.apache.org/docs/latest/multi-stage-query/index.html). For information about the endpoint specifically, see [SQL-based ingestion and multi-stage query task API](https://druid.apache.org/docs/latest/multi-stage-query/api.html).\n",
+    "\n",
+    "\n",
+    "The next cell does the following:\n",
+    "\n",
+    "- Includes a payload that inserts data from an external source into a table named wikipedia_api. The payload is in JSON format and included in the code directly. You can also store it in a file and provide the file. \n",
+    "- Saves the response to a unique variable that you can reference later to identify this ingestion task\n",
+    "\n",
+    "The example uses INSERT, but you could also use REPLACE. \n",
+    "\n",
+    "The MSQ task engine uses a task to ingest data. The response for the API includes a `taskId` and `state` for your ingestion. You can use this `taskId` to reference this task later on to get more information about it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "362b6a87",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/v2/sql/task\"\n",
+    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
+    "http_method = \"POST\"\n",
+    "\n",
+    "\n",
+    "payload = json.dumps({\n",
+    "\"query\": \"INSERT INTO wikipedia_api SELECT TIME_PARSE(\\\"timestamp\\\") \\\n",
```

Comment:
One of the scenario we need to account for is a failed ingest or users are continuing this tutorial. Adding a comment maybe highlight that you see the datasource already exist, use "REPLACE INTO" instead.